helm-chart/renku/values.yaml

```diff
@@ -83,11 +83,10 @@ global:
     # Specify which renku component should use which DB index
     dbIndex:
       gateway: "0"
-      # coreService: "1"
-      # uiServer: "2"
+      coreService: "1"
```

Comment:
that's something I have been wondering - do the different versions of the core service handle this on their own, for example by prefixing the keys with their version?

Reply:
Yes. They use: https://github.com/SwissDataScienceCenter/renku-python/blob/develop/helm-chart/renku-core/templates/deployment.yaml#L82-L83

Reply:
perfect.